docker-compose.yaml

```diff
@@ -1,9 +1,10 @@
 services:
   deye-mqtt:
     image: ghcr.io/kbialek/deye-inverter-mqtt
+    build: ./
     env_file:
       - config.env
     restart: unless-stopped
     volumes: 
-     - ./plugins:/opt/deye_inverter_mqtt/plugins:ro
-    
+      - ./plugins:/opt/deye_inverter_mqtt/plugins:ro
+      - ./data:/opt/deye_inverter_mqtt/data:rw
```

Comment:
Please revert changes in this file, as they are not relevant to the scope of the PR